### ISSUE TITLE
Exclude open tabs and save forever

### ISSUE THREAD

**Issue by AYECHESSDE** [francostellari/NppPlugins#65]
Hi, 

Can we get a simple list to select from to stop saving/exclude from saving? 

something that might be a list with checkboxes that remains in memory when you close and reopen so it always excludes this?

Many thanks for all this work!

**Comment by francostellari** [comment]
No, I'm sorry but I'm not planning to implement something like that because it's outside the purpose of the plug-in.

**Comment by AYECHESSDE** [comment]
How so? This does autosave? Excluding some of the tabs should be a relatively related feature?
